manifests/install.pp

```diff
@@ -60,9 +60,15 @@
     }
   }
 
+  if $facts['osfamily'] == 'Darwin' {
+    $root_npmrc_path = '/var/root'
```

Comment:
Is the ```root_home``` fact useful here?

Reply:
Is that a custom fact from somewhere? I'm not seeing it in any of the facter outputs for ubuntu/macos - it is probably just me though!

Reply:
@ekohl good catch!

-----

@kizzie https://github.com/puppetlabs/puppetlabs-stdlib#root_home it comes from stdlib

Reply:
Ooh I had not heard of that one - shall give it a try later!

Reply:
So I've put this in but it involves changing the test as its usually not being set to anything for the tests and so being resolved as /.npmrc